rio-config/src/defaults.rs

```diff
@@ -80,6 +80,10 @@ pub fn default_window_height() -> i32 {
     400
 }
 
+pub fn default_scroll_multiplier() -> f64 {
+    2.0
```

Comment:
Any reason to use `2.0` the default was 3.0 no?

Reply:
I think is better set it back to 3.0 to ensure have less breaking changes for this release

Reply:
rolled it back. f9d755f41fbfa3d16fdfbd7a25506f0b099b7cfb